LevelSpaceXWGUI.nlogo

```diff
@@ -572,7 +589,7 @@ to-report make-variadic-task [astring args]
   set astring add-spaces astring
   foreach string:rex-split astring "\\s" [
     ifelse member? ? args[
-      set sb lput (word "(item " (position ? args) " ?)") sb
+      set sb lput (word "(item " (position ? args) " ?" arg-num ")") sb
```

Comment:
This section of code is the first NetLogo code I've seen which would be indisputably simplified by the addition of a macro system or toolkit.